config/app.default.php

```diff
@@ -244,6 +249,8 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            
+            'url' => env('DATABASE_URL', ''),
```

Comment:
Interesting, a previous version of this pr used null and I changed it as my testing said that wasn't the way forward. I'll test this later tonight and see what I can dig up.